### PR TITLE
Bug 2188379: Backport 4.12 multisite support

### PR DESCRIPTION
**Pull request by parth-gr**
**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure)).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Comment by parth-gr** [comment]
Would be fine if I add this PR cherry-pick also
https://github.com/rook/rook/pull/11947

As most of the changes of 11947 are coming with https://github.com/red-hat-storage/rook/pull/472

File: Documentation/CRDs/Cluster/external-cluster.md [hunk 1]
```diff
@@ -56,6 +56,12 @@ python3 create-external-cluster-resources.py --rbd-data-pool-name <pool_name> --
 - `--cephfs-data-pool-name`: (optional) Provides the name of the CephFS data pool, used for creating CephFS StorageClass
 - `--rados-namespace`: (optional) Divides a pool into separate logical namespaces, used for creating RBD PVC in a RadosNamespaces
 - `--subvolume-group`: (optional) Provides the name of the subvolume group, used for creating CephFS PVC in a subvolumeGroup
+<<<<<<< HEAD
```
Comment (subhamkrai) [inline Documentation/CRDs/Cluster/external-cluster.md:59]: merge conflict?

Comment (parth-gr) [reply Documentation/CRDs/Cluster/external-cluster.md]: Thanks good catch

**Review by subhamkrai — APPROVED** [review]
LGTM

**Pull request merged**